src/components/shared/Touchable.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+import {
+  TouchableWithoutFeedbackProps,
+  Platform,
+  TouchableNativeFeedback,
+  TouchableHighlight
+} from "react-native";
+
+const Touchable: React.FunctionComponent<TouchableWithoutFeedbackProps> = ({
```

Comment:
with this, I won't be able to change ripple tint color as it won't be an acceptable prop.

Reply:
`TouchableProps = TouchableNativeFeedbackProps | TouchableHighlightProps`?

Reply:
yes